test/cluster/dtest/audit_test.py

```diff
@@ -1152,6 +1152,11 @@ def _test_insert_failure_doesnt_report_success_assign_nodes(self, session: Sessi
         all_nodes_addresses = set([node.address() for node in all_nodes])
         # the number here is arbitrary
         for i in range(256):
+            # dtest env is using FlakyRetryPolicy which has `max_retries` attribute
+            cl_three_profile = session.execution_profile_clone_update(EXEC_PROFILE_DEFAULT, consistency_level=ConsistencyLevel.THREE)
```

Comment:
I'd wrap it in some helper func, e.g. get_retries(...) and call directly instead of ` 1 + retries` in  `return audit_partition_nodes, insert_node, kill_node, stmt.query_string, 1 + retries`